Documentation/doc/biblio/geom.bib

```diff
@@ -24943,7 +24943,7 @@ @book{bm-gta-76
 
 @article{bp-adtaa-91
 , author =      "J. Bonet and J. Peraire"
-, title =       "An alternating digital tree (ADT) algortihm for $3$D geometric searching and intersection problems"
+, title =       "An alternating digital tree (ADT) algorithm for $3$D geometric searching and intersection problems"
```

Comment:
What about the`$3$D`?

Reply:
The the pair of dollars signal in LaTeX a formula and during the conversion from bibtex to a HTML page doxygen internally sees this (see in the doxygen repository the file `cite.cpp` and search for `CITE_FORMULA_`).

Note:
- it looks like that the 2 mentioned references are not used at all, as far as I know an existing bibtex database has been taken by CGAL and has not been "weeded")
- I did a small test and this confirms my remark above (example: [example.tar.gz](https://github.com/user-attachments/files/15510780/example.tar.gz) )